core/lib/core.ts

```diff
@@ -845,21 +845,29 @@ export function trackerCore(base64: boolean, callback?: (PayloadData) => void) {
 			tstamp?: Timestamp): PayloadData {
 
 			let event_schema = '';
+			let event_data = {};
 			if (schema === 'change_form'){
 				event_schema = 'iglu:com.snowplowanalytics.snowplow/change_form/jsonschema/1-0-0';
+				event_data = {
+					formId: formId,
+					elementId: elementId,
```

Comment:
Dunno if the tracker is in ES6 but if it is, in an object, when the key and the values are the same you can do like:
`{
     formId: formId,
     a: a
}`

becomes

`{
     formId,
     a
}`

Reply:
Cheers Louis, that's cleaner alright. I had hoped to get to this before my time off but am short of time - I'll address this before releasing. Appreciate the feedback!